nailgun/entities.py

```diff
@@ -8204,3 +8251,48 @@ def sync(self, synchronous=True, timeout=None, **kwargs):
         kwargs.update(self._server_config.get_client_kwargs())
         response = client.put(self.path('sync'), **kwargs)
         return _handle_response(response, self._server_config, synchronous, timeout)
+
+
+class AnsibleRoles(Entity):
+    """A representation of Ansible Roles entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._meta = {
+            'api_path': '/ansible/api/ansible_roles',
+        }
+        super().__init__(server_config, **kwargs)
+
+    def path(self, which=None):
+        """Extend ``nailgun.entity_mixins.Entity.path``.
+        The format of the returned path depends on the value of ``which``:
+
+        sync
+            /ansible_roles/sync
```

Comment:
Could you cover more AnsibleRoles endpoints like list/delete/fetch?

Reply:
I can add these, I was mostly getting a minimum viable product PR here to enable some testing I was doing in the New Host UI